docs/user/alerting/pre-configured-connectors.asciidoc

```diff
@@ -49,12 +53,35 @@ The following example shows a valid configuration 2 out-of-the box connector.
 
 [NOTE]
 ==============================================
-Sensitive properties, such as passwords, can also be stored in the {kib} keystore.
+Sensitive properties, such as passwords, can also be stored in the <<creating-keystore, {kib} keystore>>.
 ==============================================
 
+[float]
+[[preconfigured-action-type-example]]
+=== Creating a preconfigured action type
```

Comment:
This section should come first to match the order of the title "Preconfigured action types and connectors"